gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -0,0 +1,1019 @@
+                prepared_vcf_ht,
+                subsets=["gnomad"] if hgdp_1kg else SUBSETS,
+                subset_before_metric=True if hgdp_1kg else False,
+                subset_pops=parameter_dict["subset_pops"],
+                missingness_threshold=0.5,
+                verbose=args.verbose,
+            )
+
+        if args.export_vcf:
+            logger.info("Loading VCF header dict...")
+            with hl.hadoop_open(
+                release_header_path(subset="hgdp_1kg" if hgdp_1kg else None), "rb",
+            ) as f:
+                header_dict = pickle.load(f)
+
+            logger.info("Cleaning up the VCF HT for final export...")
+            prepared_vcf_ht, new_row_annots = cleanup_ht_for_vcf_export(
+                prepared_vcf_ht,
+                drop_freqs=hl.eval(prepared_vcf_ht.freq_entries_to_remove),
+                drop_hists=DROP_HISTS,
+            )
+
+            logger.info("Running check on VCF fields and info dict...")
+            if not vcf_field_check(prepared_vcf_ht, header_dict, new_row_annots):
+                logger.error("Did not pass VCF field check")
+
+            if hgdp_1kg:
+                logger.info(
+                    "Loading the HGDP + TGP subset MT and annotating with the prepared VCF HT for VCF export..."
+                )
+                t = release_subset(subset="hgdp_1kg", dense=True).mt().select_rows()
+                t = t.annotate_rows(**prepared_vcf_ht[t.row_key])
+            else:
+                t = prepared_vcf_ht
+
+            if args.test:
+                t = filter_to_test(t)
+
+            if chromosome:
+                t = hl.filter_intervals(t, [hl.parse_locus_interval(chromosome)])
+
+            logger.info(f"Exporting chromosome {chromosome}....")
+            if args.test:
+                output_path = f"{qc_temp_prefix()}gnomad.genomes_vcf_test{'hgdp_1kg_subset' if hgdp_1kg else ''}.vcf.bgz"
+            else:
+                output_path = release_vcf_path(
+                    contig=chromosome, subset="hgdp_1kg_subset" if hgdp_1kg else None,
+                )
+
+            hl.export_vcf(
+                rekey_new_reference(t, export_reference),
+                output_path,
+                append_to_header=append_to_vcf_header_path(
+                    "hgdp_1kg" if hgdp_1kg else ""
+                ),
+                metadata=header_dict,
+                tabix=True,
+            )
+
+    finally:
+        logger.info("Copying hail log to logging bucket...")
+        hl.copy_log(f"{qc_temp_prefix()}logs/vcf_export.log")
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--hgdp_1kg_subset",
+        help="Use HGDP + TGP subset Matrix table instead of the gnomAD release HT",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--test",
+        help="Create release files using only 5 partitions on chr20, chrX, and chrY for testing purposes",
```

Comment:
```suggestion
        help="Create release files using only 2 partitions on chr20, chrX, and chrY for testing purposes",
```
the default for `filter_to_test` is 2 partitions